### PR TITLE
Baskets instead of pruners #21

### PR DESCRIPTION
**Pull request by rodricios**
This also reduces ``article.py`` (aka eatiht)

### CODE REVIEW

File: libextract/coretools.py [hunk 1]
```diff
@@ -54,6 +54,15 @@ def mapper(etree):
     return decorator
 
 
+def get_node(pair):
+    """
```
Comment (eugene-eeo) [inline libextract/coretools.py]: I am a little hesitant on leaving this in coretools, it is more implementation specific to article's strategy than generalised.

Comment (rodricios) [inline libextract/coretools.py]: Pretend article.py didn't exist :P

Comment (rodricios) [inline libextract/coretools.py]: I do agree that it could exist somewhere outside of coretools.py too.

Comment (eugene-eeo) [inline libextract/coretools.py]: Utils/helpers.py?

**Comment by rodricios** [comment]
For now, sure, I think that'd be a good place :)

**Pull request merged**